Examples/generate-app.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Copyright (c) 2015-present, Microsoft Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+# The goal of this script is to automate the testing of react-native, react-native-code-push and 
+# their internal compatibility. Run **./run.sh** to create an app and test it. 
```

Comment:
I think this would be better described as a script that can generate a sample app for a specific version of React Native and CodePush. Also, it is no longer named `run.sh`